src/js/ng-suggestion.js

```diff
@@ -65,8 +65,14 @@
         this.keydownHandler = function(input) {
           return ($event) => {
             var currentOption = input.dropdown.currentOption;
-            if (currentOption && $event.keyCode === 13) { // Enter
+            var freeText = input.freeText;
+            var enterAction = input.enterAction;
```

Comment:
Are these extra variables necessary? Couldn't we use `input.freeText` and `input.enterAction` directly?